sktime/clustering/partitioning/_cluster_approximations.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+"""Approximations for time series clusterers"""
+
+__author__ = ["Christopher Holder", "Tony Bagnall"]
+__all__ = ["Medoids"]
+
+import numpy as np
+
+from sktime.clustering.base._typing import NumpyArray, MetricFunction
+from sktime.clustering.base.base import BaseApproximate
+from sktime.utils.clustering_utils import compute_pairwise_distances
+
+
+class Medoids(BaseApproximate):
+    """Medoids Approximate
+
+    Parameters
+    ----------
+    series: Numpy_Array
+        series to perform approximation on
+
+    """
+
+    def __init__(self, series: NumpyArray, metric: MetricFunction):
+        super(Medoids, self).__init__(series)
+        self.metric = metric
+
+    def approximate(self) -> int:
```

Comment:
```suggestion
    def _approximate(self) -> int:
```

Reply:
This must have got readded by mistake when i was fixing whatever happened with the pr the other day good spot

Reply:
Same here want this to be a publically avaliable method